gavel/templates/admin_annotator.html

```diff
@@ -28,6 +28,10 @@
         <td>{{ annotator.secret }}</td>
```

Comment:
I guess it is redundant. I left it there because I didn't know what you intended to use that row for.